Test/logger/ProofDependencyLogging.dfy

```diff
@@ -0,0 +1,424 @@
+// RUN: %baredafny verify --log-format:text --boogie -trackVerificationCoverage "%s" > "%t"
```

Comment:
I'm still a little puzzled on the UX.

One use-case that's clear to me is giving warnings whenever code is not used by any assertion batch. For that I wouldn't use this report, but instead turn on a warning `--warn-unused-by-verification`

Can you give an example of how how the information presented in this report would be used?

Reply:
Yeah, this isn't really meant to be something that would be used by anyone other than the Dafny developers in its current state. It could be, but would require some expertise and manual effort. I'm building a more useful set of UI mechanisms on top of it, but I want to merge this now to get things done in pieces so there's not one monster PR to review. :) Think of it similarly to the type system refresh @RustanLeino  is doing.

Reply:
Okay, that works.